proto/cosmos/base/tendermint/v1beta1/query.proto

```diff
@@ -87,10 +86,10 @@ message GetLatestValidatorSetResponse {
 
 // Validator is the type for the validator-set.
 message Validator {
-  string              address           = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
-  google.protobuf.Any pub_key           = 2;
-  int64               voting_power      = 3;
-  int64               proposer_priority = 4;
+  string                       address           = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  .tendermint.crypto.PublicKey pub_key           = 2 [(gogoproto.nullable) = false];
```

Comment:
why this change?

Reply:
Because `google.protobuf.Any` will use the sdk's codec package, which is unnecessary here if we use the original types

Reply:
ah true, dam that type. It was moved now but i guess the code gen needs to be updated